content/en/docs/concepts/configuration/secret.md

```diff
@@ -524,6 +524,12 @@ secret "prod-db-secret" created
 $ kubectl create secret generic test-db-secret --from-literal=username=testuser --from-literal=password=iluvtests
 secret "test-db-secret" created
 ```
+{{< note >}}
+**Note:** Special characters such as `$`, `\*`, and `!` require escaping.
```

Comment:
This escaping is only required for --from-literal, right? I guess this is because Cobra lib parsing requires escaping.
In terms of --from-file, we don't need escaping, right?
If so, we should clarify it.